### PR TITLE
Update readme with link to breaking changes page of website.

### PR DESCRIPTION
**Pull request by christopherfujino**
Explicitly call out the breaking changes page of the website in the README.

### CODE REVIEW

**Review by timsneath — APPROVED** [review]
Great!

File: README.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ and open source.
 
 For announcements about new releases and breaking changes, follow the
 [flutter-announce@googlegroups.com](https://groups.google.com/forum/#!forum/flutter-announce)
-mailing list.
+mailing list or see the
+[breaking changes](https://flutter.dev/docs/release/breaking-changes) page of
+Flutter's official website.
```
Comment (timsneath) [inline README.md:23]: I'd probably remove the 'of Flutter's official website' clause. End the sentence at 'page'.

Comment (christopherfujino) [reply README.md]: Thanks, done.

**Comment by Hixie** [comment]
LGTM. I'd also be happy with us removing the mention of the mailing list, but that may be a bigger discussion.

**Pull request merged**